web/src/title/loader.ts

```diff
@@ -47,12 +48,16 @@ export async function createNewProject(
 
 export async function loadFromLocalStorage(key: string) {
   currentProjectKey.set(key);
+  let isCnt = key.startsWith("ltn_cnt/");
+  // TODO Should we also change the URL?
```

Comment:
Should we rewrite history and set index.html or cnt.html?

Reply:
I'll get to this tomorrow; I'll merge but keep #195 open.

I like the `cnt/` subdirectory vs `cnt.html`, do we want to go with that too? Should be a pretty easy change in vite config and making a new cnt/index.html.

Reply:
(It's not challenging I don't think, just late / open PRs weigh on my mind a bit)

Reply:
> I like the cnt/ subdirectory vs cnt.html, do we want to go with that too?

Sure! 

TBH in my mind they are semantically equivalent, except that the `.html` communicates that your website has been online since the Clinton administration.